### PR TITLE
[12.0][IMP] sale_discount_display_amount, add discount amount on document report

### PR DESCRIPTION
**Pull request by GSLabIt**

### CODE REVIEW

File: sale_discount_display_amount/__manifest__.py [hunk 1]
```diff
@@ -6,16 +6,18 @@
     'summary': """
         This addon intends to display the amount of the discount computed on
         sale_order_line and sale_order level""",
-    'version': '12.0.1.0.0',
+    'version': '12.0.1.0.1',
```
Comment (rousseldenis) [inline sale_discount_display_amount/__manifest__.py]: No need to change this, will be done on merge

**Review by rousseldenis — CHANGES_REQUESTED** [review]

**Review by SicurSam — APPROVED** [review]

**Review by HaraldPanten — APPROVED** [review]
LGTM after Denis's requested changes

File: sale_discount_display_amount/__manifest__.py (hunk 1 above)
Comment (GSLabIt) [reply sale_discount_display_amount/__manifest__.py]: ok, will do

**Comment by GSLabIt** [comment]
@HaraldPanten @rousseldenis done. Seems Travis is stuck

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**